src/lib.rs

```diff
@@ -1172,7 +1358,7 @@ impl Context {
     ///////////////////////////////////////////////////////////////////////////
 
     /// Returns a mutable reference to the native ESYS context handle.
-    fn mut_context(&mut self) -> *mut ESYS_CONTEXT {
+    pub fn mut_context(&mut self) -> *mut ESYS_CONTEXT {
```

Comment:
Why is this necessary?

The question regarding exposing the esys context have been discussed previously(#50, #53). The reason for not doing was to not leak the state of the context. See @ionut-arm response in #53 .

Reply:
Sorry, that was a test from me that I forgot to undo.